new/LIT7088QeneTawa.xml

```diff
@@ -0,0 +1,90 @@
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Qǝne</foreign> without type indication in the manuscript. It has nine lines, meaning
+                    that it can be classified as a <foreign xml:lang="gez">mawaddəs qəne</foreign> 
+                    (possibly of the subtype <foreign xml:lang="gez">kʷəlləkəmu</foreign>) according to
+                    <bibl><ptr target="bm:Guidi1900Qene"/></bibl>. 
+                    It is attributed to <persName ref="PRS14574GabraLeul" cert="high" corresp="SD">Gabra Ləʿul</persName> 
+                    in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Mawaddes" cert="medium"/>
+                    <term key="Kwellekomu" cert="medium"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-10-04">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7088QeneTawa" passive="UppQ1524o"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7088QeneTawa" passive="PRS14574GabraLeul"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="UppQ1524o"/> on fols 14r, 15r. Division into verses is indicated with ፤ .
+                    The end of the poem is indicated with ። .</note>
+                <ab>
+                    <l n="1">ጣዕዋ፡ ንኡስ፡ መድኀ<choice resp="JK"><sic>ኒ</sic><corr>ኔ</corr></choice><supplied reason="omitted">፡</supplied>
+                        አለም፡ ዘአሰረከ፡ እግረከ፡ ኖላዌ፡ እንስሳ፡ ቀኖት፤</l>
+                    <l n="2">አድያሜ፡ ዮርዳኖስ፡ አፉከ፡ እንተ፡ አልቦቱ፡ ሐሰት፤</l>
+                    <l n="3">አደ፡ ንዋ፡ ዮሐንስ፡ ሐሞት፤</l>
+                    <l n="4">ወአፃ<choice resp="JK"><sic>ቢ</sic><corr>ብ</corr></choice><choice resp="JK"><sic>እ</sic><corr>ኢ</corr></choice>ቢእከ፡ 
```

Comment:
Perhaps it is easier for the user to get it, if you have one sic and one corr for both letters: 
`ወአፃ<choice resp="JK"><sic>ቢእ</sic><corr>ብኢ</corr></choice>ከ፡`

One ቢእ following the markup is redundant in the end of the line.